### PR TITLE
Binding quickstart - Cron input PostgreSQL output quickstart doc

### PR DESCRIPTION
**Pull request by nirmash**
Signed-off-by: Nir Mashkowski <nimashkowski@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Doc for new Bindings Quickstart

### CODE REVIEW

File: daprdocs/content/en/getting-started/quickstarts/_index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Hit the ground running with our Dapr quickstarts, complete with code samples aim
 | [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) |  Call a method on another service using the service invocation API. |
 | [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Send and receive messages using the publish and subscribe API. |
 | [State Management]({{< ref statemanagement-quickstart.md >}}) | Create stateful applications using the state management API. |
-| Bindings           | Coming soon. |
+| [Bindings]({{< ref bindings-quickstart.md >}}) | Use Dapr's input and output bindings to schedule a database insert job. |
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/_index.md]:
```suggestion
| [Bindings]({{< ref bindings-quickstart.md >}}) | Schedule a database insert job using the input and output bindings API. |
```

To match the sentence structure of the other quickstart descriptions.

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,778 @@
+---
+type: docs
+title: "Quickstart: Bindings"
+linkTitle: "Dapr Bindings"
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
linkTitle: "Bindings"
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,778 @@
...
+weight: 70
+description: "Get started with Dapr's Binding building block"
+---
+
+Let's take a look at Dapr's [Binding building block]({{< ref bindings >}}). In this Quickstart, you will schedule a script to run every 10 seconds using an input [Cron](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/) binding. The script will process a Json file and output data to a SQL database using the [PostgreSQL](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres) Dapr binding.
+
+Using bindings, you can trigger your app with events coming in from external systems, or interface with external systems. Bindings provide several benefits for you and your code:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Using bindings, you can:

- Trigger your app with events coming in from external systems.
- Interface with external systems. 
```

To keep the quickstart streamlined and get people straight into the example, let's leave the following list out. We link to the bindings overview, where they can find this list of benefits.

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,778 @@
...
+ - Remove the complexities of connecting to, and polling from, messaging systems such as queues and message buses.
+ - Focus on business logic and not implementation details of how to interact with a system.
+ - Keep your code free from SDKs or libraries.
+ - Handle retries and failure recovery.
+ - Switch between bindings at run time.
+ - Build portable applications where environment-specific bindings are set-up and no code changes are required.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]: Per comment above, remove lines 12 - 17.

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,778 @@
...
+
+For a specific example, bindings would allow your microservice to respond to incoming Twilio/SMS messages without adding or configuring a third-party Twilio SDK, worrying about polling from Twilio (or using websockets, etc.).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]: I'd remove this as well - it doesn't really speak to this specific quickstart example, and it's already in the bindings overview.

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,778 @@
...
+```
+Connect to the new database and create the `orders` table.
+```bash
+\c orders;
+create table orders ( orderid int, customer text, price float ); select * from orders;
+```
+
+### Step 4: Schedule a Cron job and write to the database
+
+In a terminal window, navigate to the `sdk` directory.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In a new terminal window, navigate to the `sdk` directory.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 7]
```diff
@@ -0,0 +1,778 @@
...
+```bash
+pip3 install -r requirements.txt
+```
+
+Run the `python-quickstart-binding-sdk` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id python-quickstart-binding-sdk --app-protocol grpc --app-port 50051 --components-path ../../components python3 batch.py
+```
+The `python-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer` and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `python-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 8]
```diff
@@ -0,0 +1,778 @@
...
+        print(json.dumps(payload), flush=True)
+        try:
+            d.invoke_binding(binding_name=sql_binding, operation='exec', binding_metadata=payload, data='')        
+        except Exception as e:
+            print(e, flush=True)
+```
+
+### Step 5: View the Output Binding log
+
+Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer` and `Price` as a payload.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer`, and `Price` as a payload.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 9]
```diff
@@ -0,0 +1,778 @@
...
+Output Binding `print` statement output:
+```
+== APP == {"sql": "insert into orders (orderid, customer, price) values (1, 'John Smith', 100.32);"}
+== APP == {"sql": "insert into orders (orderid, customer, price) values (2, 'Jane Bond', 15.4);"}
+== APP == {"sql": "insert into orders (orderid, customer, price) values (3, 'Tony James', 35.56);"}
+```
+
+### Step 6: Process the `orders.json` file on a schedule
+
+The `python-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a json file containing order information.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `python-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a JSON file containing order information.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 10]
```diff
@@ -0,0 +1,778 @@
+---
+type: docs
+title: "Quickstart: Bindings"
+linkTitle: "Dapr Bindings"
+weight: 70
+description: "Get started with Dapr's Binding building block"
+---
+
+Let's take a look at Dapr's [Binding building block]({{< ref bindings >}}). In this Quickstart, you will schedule a script to run every 10 seconds using an input [Cron](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/) binding. The script will process a Json file and output data to a SQL database using the [PostgreSQL](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres) Dapr binding.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Let's take a look at Dapr's [Binding building block]({{< ref bindings >}}). In this Quickstart, you will schedule a script to run every 10 seconds using an input [Cron](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/) binding. The script will process a JSON file and output data to a SQL database using the [PostgreSQL](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres) Dapr binding.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 11]
```diff
@@ -0,0 +1,778 @@
...
+
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Run the [PostgreSQL instance](https://www.postgresql.org/) in a Docker container on your machine.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 12]
```diff
@@ -0,0 +1,778 @@
...
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Run the [PostgreSQL instance](https://www.postgresql.org/) in a Docker container on your machine.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 13]
```diff
@@ -0,0 +1,778 @@
...
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
+
+To run the container locally, run:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 14]
```diff
@@ -0,0 +1,778 @@
...
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
+
+To run the container locally, run:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 15]
```diff
@@ -0,0 +1,778 @@
...
+```
+Connect to the new database and create the `orders` table.
+```bash
+\c orders;
+create table orders ( orderid int, customer text, price float ); select * from orders;
+```
+
+### Step 4: Schedule a Cron job and write to the database
+
+In a terminal window, navigate to the `sdk` directory.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In a new terminal window, navigate to the `sdk` directory.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 16]
```diff
@@ -0,0 +1,778 @@
...
+```bash
+npm install
+```
+
+Run the `javascript-quickstart-binding-sdk` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id javascript-quickstart-binding-sdk --app-port 3500 --dapr-http-port 5051 node batch.js --components-path ../../components
+```
+The `javascript-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer` and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `javascript-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 17]
```diff
@@ -0,0 +1,778 @@
...
+        });
+        console.log('Finished processing batch');
+      });
+    return 0;
+}
+```
+
+### Step 5: View the Output Binding log
+
+Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer` and `Price` as a payload.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer`, and `Price` as a payload.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 18]
```diff
@@ -0,0 +1,778 @@
...
+== APP == [Dapr-JS] Server Started
+== APP == {  "sql": "insert into orders (orderid, customer, price) values (1, 'John Smith', 100.32);" }
+== APP == {  "sql": "insert into orders (orderid, customer, price) values (2, 'Jane Bond', 15.4);" }
+== APP == {  "sql": "insert into orders (orderid, customer, price) values (3, 'Tony James', 35.56);" }
+== APP == Finished processing batch
+```
+
+### Step 6: Process the `orders.json` file on a schedule
+
+The `javascript-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a json file containing order information.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `javascript-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a JSON file containing order information.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 19]
```diff
@@ -0,0 +1,778 @@
...
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Run the [PostgreSQL instance](https://www.postgresql.org/) in a Docker container on your machine.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 20]
```diff
@@ -0,0 +1,778 @@
...
+
+To run the container locally, run:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 21]
```diff
@@ -0,0 +1,778 @@
...
+```
+Connect to the new database and create the `orders` table.
+```bash
+\c orders;
+create table orders ( orderid int, customer text, price float ); select * from orders;
+```
+
+### Step 4: Schedule a Cron job and write to the database
+
+In a terminal window, navigate to the `sdk` directory.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In a new terminal window, navigate to the `sdk` directory.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 22]
```diff
@@ -0,0 +1,778 @@
...
+dotnet restore
+dotnet build batch.csproj
+```
+
+Run the `csharp-quickstart-binding-sdk` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id csharp-quickstart-binding-sdk --app-port 7001 --components-path ../../components -- dotnet run --project batch.csproj
+```
+The `csharp-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer` and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `csharp-quickstart-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 23]
```diff
@@ -0,0 +1,778 @@
...
+        sqlText}
+    };
+    await client.InvokeBindingAsync(sqlBindingName, "exec", command,command);
+    Console.WriteLine(sqlText);
+}
+```
+
+### Step 5: View the Output Binding log
+
+Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer` and `Price` as a payload.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer`, and `Price` as a payload.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 24]
```diff
@@ -0,0 +1,778 @@
...
+Output Binding `Console.WriteLine` statement output:
+```
+== APP == insert into orders (orderid, customer, price) values (1, 'John Smith', 100.32);
+== APP == insert into orders (orderid, customer, price) values (2, 'Jane Bond', 15.4);
+== APP == insert into orders (orderid, customer, price) values (3, 'Tony James', 35.56);
+```
+
+### Step 6: Process the `orders.json` file on a schedule
+
+The `csharp-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a json file containing order information.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `csharp-quickstart-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a JSON file containing order information.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 25]
```diff
@@ -0,0 +1,778 @@
...
+
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+### Step 2: Run PostgreSQL Docker Container Locally
+
+In order to run the PostgreSQL bindings quickstart locally, you will run the [PostgreSQL instance](https://www.postgresql.org/) in a docker container on your machine.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Run the [PostgreSQL instance](https://www.postgresql.org/) in a Docker container on your machine.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 26]
```diff
@@ -0,0 +1,778 @@
...
+```
+Connect to the new database and create the `orders` table.
+```bash
+\c orders;
+create table orders ( orderid int, customer text, price float ); select * from orders;
+```
+
+### Step 4: Schedule a Cron job and write to the database
+
+In a terminal window, navigate to the `sdk` directory.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In a new terminal window, navigate to the `sdk` directory.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 27]
```diff
@@ -0,0 +1,778 @@
...
+```bash
+go build batch.go
+```
+
+Run the `go-input-binding-sdk` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id go-input-binding-sdk --app-port 6002 --dapr-http-port 6003 --dapr-grpc-port 60002 go run batch.go --components-path ../../components
+```
+The `go-input-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer` and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `go-input-binding-sdk` uses the PostgreSQL Output Binding defined in the [`bindings.yaml`]({{< ref "#bindingsyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. This code is executed every 10 seconds (defined in [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) in the `components` directory).
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 28]
```diff
@@ -0,0 +1,778 @@
...
+	if err != nil {
+		return err
+	}
+	return nil
+}
+```
+
+### Step 5: View the Output Binding log
+
+Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer` and `Price` as a payload.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer`, and `Price` as a payload.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 29]
```diff
@@ -0,0 +1,778 @@
...
+== APP == dapr client initializing for: 127.0.0.1:60002
+== APP == insert into orders (orderid, customer, price) values (1, 'John Smith', 100.32);
+== APP == insert into orders (orderid, customer, price) values (2, 'Jane Bond', 15.40);
+== APP == insert into orders (orderid, customer, price) values (3, 'Tony James', 35.56);
+== APP == Finished processing batch
+```
+
+### Step 6: Process the `orders.json` file on a schedule
+
+The `go-input-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a json file containing order information.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
The `go-input-binding-sdk` uses the Cron Input Binding defined in the [`cron.yaml`]({{< ref "#cronyaml-component-file" >}}) component to process a JSON file containing order information.
```

**Review by hhunter-ms — DISMISSED** [review]
Just a few grammar and consistency updates. I ran through the quickstart and all the languages worked for me!

**Comment by hhunter-ms** [comment]
Issue #2477

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 30]
```diff
@@ -0,0 +1,825 @@
...
+weight: 70
+description: "Get started with Dapr's Binding building block"
+---
+
+Let's take a look at Dapr's [Bindings building block]({{< ref bindings >}}). Using bindings, you can:
+
+- Trigger your app with events coming in from external systems.
+- Interface with external systems.
+
+In this Quickstart, you will schedule a batch script to run every 10 seconds using an input [Cron](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/) binding. The script will process a JSON file and output data to an external SQL database using the [PostgreSQL](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres) Dapr binding.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In this Quickstart, you will schedule a batch script to run every 10 seconds using an input [Cron](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/) binding. The script processes a JSON file and outputs data to a SQL database using the [PostgreSQL](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres) Dapr binding.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 31]
```diff
@@ -0,0 +1,825 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 32]
```diff
@@ -0,0 +1,825 @@
...
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+### Step 2: Run PostgreSQL Docker container locally
+
+You will run the [PostgreSQL instance](https://www.postgresql.org/) locally in a Docker container on your machine. The Quickstart sample includes a Docker Compose file to locally customize, build, run, and initialize the `postgres` container with a default `orders` table.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
You run the [PostgreSQL instance](https://www.postgresql.org/) locally in a Docker container on your machine. The Quickstart sample includes a Docker Compose file to locally customize, build, run, and initialize the `postgres` container with a default `orders` table.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 33]
```diff
@@ -0,0 +1,825 @@
...
+
+In a terminal window, from the root of the Quickstarts clone directory, navigate to the `bindings\db` directory.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
In a terminal window, from the root of the Quickstarts clone directory, navigate to the `bindings/db` directory.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 34]
```diff
@@ -0,0 +1,825 @@
...
+                                binding_metadata=payload, data='')
+        return resp
+    except Exception as e:
+        print(e, flush=True)
+        raise SystemExit(e)
+```
+
+### Step 4: View the output of the job
+
+Notice, as specified above, the code invokes the Output Binding with the `OrderId`, `Customer`, and `Price` as a payload.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Notice, as specified above, the code invokes the output binding with the `OrderId`, `Customer`, and `Price` as a payload.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 35]
```diff
@@ -0,0 +1,825 @@
...
+       1 | John Smith | 100.32
+       2 | Jane Bond  |   15.4
+       3 | Tony James |  35.56
+```
+
+#### `components\binding-cron.yaml` component file
+
+When you execute the `dapr run` command and specify the component path, the Dapr sidecar:
+
+- Initiates the Cron [Binding building block]({{< ref bindings >}})
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
- Initiates the Cron [binding building block]({{< ref bindings >}})
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 36]
```diff
@@ -0,0 +1,825 @@
...
+  namespace: quickstarts
+spec:
+  type: bindings.cron
+  version: v1
+  metadata:
+  - name: schedule
+    value: "@every 10s" # valid cron schedule
+```
+
+**Note:** The `metadata` section of `binding-cron.yaml` contains a [Cron expression](/reference/components-reference/supported-bindings/cron/) that specifies how often the binding will be invoked.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
**Note:** The `metadata` section of `binding-cron.yaml` contains a [Cron expression](/reference/components-reference/supported-bindings/cron/) that specifies how often the binding is invoked.
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 37]
```diff
@@ -0,0 +1,825 @@
...
+
+#### `component\bindings-postgres.yaml` component file
+
+When you execute the `dapr run` command and specify the component path, the Dapr sidecar:
+
+- Initiates the PostgreSQL [Binding building block]({{< ref /reference/components-reference/supported-bindings/postgres.md >}})
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
- Initiates the PostgreSQL [binding building block]({{< ref /reference/components-reference/supported-bindings/postgres.md >}})
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 38]
```diff
@@ -0,0 +1,825 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Latest Node.js installed](https://nodejs.org/download/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 1: Set up the environment
+
+Clone the [sample we've provided in our Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings).
```

**Comment by msfussell** [comment]
@hhunter-ms - I reviewed the first python code section and hopefully you can apply those suggestion to the other languages in the QS

**Comment by greenie-msft** [comment]
@nirmash, does running this branch locally work for you? When I try, I get the error: 

/bindings-quickstart.md:391:62": failed to extract shortcode: shortcode "reference" has no .Inner, yet a closing tag was provided

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 39]
```diff
@@ -0,0 +1,1050 @@
...
+  namespace: quickstarts
+spec:
+  type: bindings.cron
+  version: v1
+  metadata:
+  - name: schedule
+    value: "@every 10s" # valid cron schedule
+```
+
+**Note:** The `metadata` section of `binding-cron.yaml` contains a [Cron expression]({{ ref cron.md }}) that specifies how often the binding is invoked.
```
Comment (greenie-msft) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]: I think this link is missing the "<" ">"

**Comment by msfussell** [comment]
@greenie-msft @hhunter-ms  - Are these PPT image files checked in somewhere?

**Comment by greenie-msft** [comment]
> @greenie-msft @hhunter-ms - Are these PPT image files checked in somewhere?

I don't think they are but we can check them into the diagrams ppt?

**Review by greenie-msft — APPROVED** [review]

**Review by msfussell — APPROVED** [review]
Fantastic to see this complete!

**Pull request merged**